getting-started/support_matrix.md

```diff
@@ -29,6 +29,7 @@ KubeArmor supports following types of workloads:
 | Rancher    | [K3S] | [Distros] | x86_64 | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | [BPFLSM], AppArmor |
 | Oracle     | [Ampere] | [UEK] | ARM | :heavy_check_mark: | :heavy_check_mark: | :x: | :heavy_check_mark: | :heavy_check_mark: | [1084] |
 | VMWare     | [Tanzu] | TBD | x86_64 | :construction: | :construction: | :construction: | :construction: | :construction: | [1064] |
+| Mirantis     | [k8s] | Ubuntu>- 20.04 | x86_64 | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | [1181] |
```

Comment:
Please mention that it's tested with AppArmor

Reply:
Added the Document in the Issues and also completed the changes that Ankur mentioned it.